package.json

```diff
@@ -69,7 +69,7 @@
     "supertest": "^3.1.0"
   },
   "engines": {
-    "node": "^4.8.4 || ^6.10.1 || ^7.10.1 || >= 8.1.4"
```

Comment:
We should break the promise that we support these node versions in current version even they were EOF. we can change this in the next major version but not now.

Reply:
@dead-horse you can not do anything. you don't know when to release V3 too.
@3imed-jaberi can we together rewrite koa with LTS and current node? 😄

Reply:
@anlexN Koa is MIT licensed so you are free to fork and do what ever you want with it. However, please be respectful to all contributors. We (Koa maintainers of different ranks) are in agreement that a new version of Koa isn't warranted unless there's significant changes/improvements to the library. You may disagree with the choices made - but that's no reason to be disrespectful. You started writing tickets a few days ago, we've been maintaining Koa for years.

Most of the things you've raised are breaking changes. We take semver very seriously here and have a rather large ecosystem to consider before introducing breakage. There will always be things to improve, we are always open to debate changes, PRs are always welcome. 

An underlined image with a passive aggressive comment isn't fun for us. Keep it respectful and joyful for all parties involved. A happy ticket, and preferably PR, is more likely to be taken seriously.

Reply:
can we make a new branch focused on new idea? i need this branch very much, i hope many people like it too, and so many pull request sound very wonderful but you don't like merge.

Reply:
You can fork the repo and create as many branches as you need.

Reply:
i hope you make, you are official, please?

Reply:
@3imed-jaberi please review [anlexN/koa](https://github.com/anlexN/koa), we can together do it.

Reply:
Hi @dead-horse, @miwnwski, @anlexN 👋 .. 
Sorry to be absent for a month due to my graduation ... I believe that working on the next release (koa@3.x.x) and planning for it with somethings like improve perf. is more important than rewrite with typescript. !!